buildtools/composer/composer_test.go

```diff
@@ -46,3 +61,31 @@ func TestNoDependenciesNamedPHP(t *testing.T) {
 		}
 	}
 }
+
+func TestComposer_Show(t *testing.T) {
```

Comment:
I thought you might bring up testing of this package in general—now that invocations of `show` and `install` are mockable.

Do we not need to test if the php analyzer is correctly determined to be the one to use?:
```
a, err := analyzers.New(m)
assert.NoError(t, err)
assert.IsType(t, &php.Analyzer{}, a)
```

Reply:
To clarify here, because I saw you remove this line in the test, I think you are asking if we do not need to test this case anymore? I would agree that it seems like overkill and doesn't add much to the strength of the tests.